event.html

```diff
@@ -59,34 +59,34 @@
     <div class="container">
         <div class="jumbotron event-jumbo mt-5 rounded">
 
-            <h2 class="p-2">
+            <h2 class="p-3">
                 SNBRN pool side at<br>phoenix hotel
             </h2>
             <img src="IMG/SNBRN.jpeg" class="h-50 mx-auto d-block">
-            <button type="button" class="btn btn-light col-2 m-4 d-flex justify-content-sm-center align-self-end">buy tickets</button>
+            <a class="btn btn-light col-2 m-4 p-3 d-flex justify-content-sm-center align-self-end text-uppercase "href="https://www.eventbrite.com/e/summertime-radness-dj-seinfeld-tickets-314960684887" target="_blank">buy tickets</a>
         </div>
 
         <div class="jumbotron event-jumbo mt-5">
-            <h2 class="p-2">
+            <h2 class="p-3">
                 wax motif poolside at<br>phoenix hotel
             </h2>
             <img src="IMG/Wax-motif.jpeg" class=" h-50 mx-auto d-block ">
-            <button type="button" class="btn btn-light col-2 m-4 d-flex justify-content-sm-center align-self-end ">buy tickets</button>
+            <a class="btn btn-light col-2 m-4 p-3 d-flex justify-content-sm-center align-self-end text-uppercase "href="https://www.eventbrite.com/e/summertime-radness-dj-seinfeld-tickets-314960684887" target="_blank">buy tickets</a>
```

Comment:
Hm, it also seems a little strange to put `col-2` on an anchor tag. I'd have to see what this looks like but I'd expect something more like:
```
<div class="col-2">
  <a class="btn....etc">
</div>
```
But then again I'm not sure what you're going for layout wise here.